apps/pdc-frontend/src/styles/globals.css

```diff
@@ -195,6 +195,25 @@ p:has(a[class="utrecht-skip-link utrecht-skip-link--visible-on-focus"]) {
   padding-inline-end: var(--utrecht-navigation-toggle-button-padding-end);
 }
 
+/* openforms datalist */
+.openforms-card {
+  --of-card-padding-block-end: 0;
+  --of-card-padding-block-start: 0;
+  --of-card-padding-inline-end: 0;
+  --of-card-padding-inline-start: 0;
+}
+.openforms-progress-indicator {
+  --of-progress-indicator-padding-block-end: 0;
+}
+
+.utrecht-data-list--openforms > .utrecht-data-list__item:first-of-type {
+  margin-block-start: 0;
+}
+
+.utrecht-data-list--openforms > .utrecht-data-list__item {
+  margin-block-start: var(--utrecht-data-list-rows-item-margin-block-start);
+}
```

Comment:
Zou dit ook simpeler kunnen?

```
.utrecht-data-list--openforms {
   --utrecht-data-list-item-margin-block-start: var(--utrecht-data-list-rows-item-margin-block-start);
 }
```

Reply:
dit kan niet want die utrecht token bestaad volgens niet in de form